SystemServices/System.json

```diff
@@ -763,8 +834,8 @@
                 ]
             }
         },
-        "getPreviousRebootInfo2":{
-            "summary": "Returns detailed information about a reboot",
+        "getPreviousRebootInfo":{
```

Comment:
The API name is actually getPreviousRebootInfo2. This exists for all versions along with getPreviousRebootInfo.

Reply:
updated the method signature accordingly.